### ISSUE TITLE
Geräteversatz vertikal

### ISSUE THREAD

**Issue by Noggele** [Courseplay/Courseplay_FS22#1419]
Wo ist denn der vertikale Geräteversatz hin?
Meine Vorschneider bei den Weinreben kollidieren nun mit den Rebstöcken. 

Vorher konnte ich den Versatz noch einstellen und es hat funktioniert !!!

![Geräteversatz](https://user-images.githubusercontent.com/101008421/163878116-ad31242d-b100-4f5a-b0f2-5f73daec4e48.jpg)

**Comment by TobiHH78** [comment]
Dürfte hier sein...

https://github.com/Courseplay/Courseplay_FS22/releases/tag/7.0.1.12

Added an "expert" Mode, wich we contain all the settings you are used to. However, the expert mode is turned off by default.
We try to make courseplay more accessible for new players to not be scared off all the settings.
Settings that are not accessible when expert mode is off, will use their default values.
**To activate the expert Mode, go into global settings and turn it on.**

**Comment by Tensuko** [comment]
Nein. Der wurde entfernt. Dein Problem muss anders gelöst werden. Es ist schon eher schade, dass es wieder auftritt.

Und wenn du das bug template nutzt fülle es bitte aus ider nutze question.

**Comment by Tensuko** [comment]
Das Wenden funktioniert wunderbar und ein vertikaler Offset ist nicht nötig.
![fsScreen_2022_05_03_20_04_01](https://user-images.githubusercontent.com/27344744/166514933-c3452104-99ca-41df-acec-dde11158d31a.png)
